composeApp/src/commonMain/kotlin/com/overdrive/cruiser/views/LoginView.kt

```diff
@@ -1,28 +1,191 @@
+                        .padding(24.dp),
+                ) {
+                    Button(
+                        onClick = { onGoogleClick() },
+                        shape = RoundedCornerShape(12.dp),
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .height(55.dp)
+                            .shadow(elevation = 8.dp, shape = RoundedCornerShape(12.dp))
+                            .background(color = Color.White),
+                        colors = ButtonDefaults.buttonColors(
+                            backgroundColor = Color.White,
+                        ),
+                    ) {
+                        Box(
+                            modifier = Modifier.fillMaxWidth(),
+                            contentAlignment = Alignment.Center
+                        ) {
+                            Icon(
+                                painter = painterResource(Res.drawable.google_logo),
+                                contentDescription = null,
+                                tint = Color.Unspecified,
+                                modifier = Modifier
+                                    .align(Alignment.CenterStart)
+                                    .size(50.dp),
+                            )
+                            Text(
+                                text = "Continue With Google",
+                                color = contentColorFor(Color.Black),
+                            )
+                        }
+                    }
+
+                    Spacer(modifier = Modifier.height(16.dp))
+
+                    Button(
+                        onClick = { onGoogleClick() },
+                        shape = RoundedCornerShape(12.dp),
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .height(55.dp)
+                            .shadow(elevation = 8.dp, shape = RoundedCornerShape(12.dp))
+                            .background(color = Color.White),
+                        colors = ButtonDefaults.buttonColors(
+                            backgroundColor = Color.White,
+                        ),
+                    ) {
+                        Box(
+                            modifier = Modifier.fillMaxWidth(),
+                            contentAlignment = Alignment.Center
+                        ) {
+                            Icon(
+                                painter = painterResource(Res.drawable.apple_logo),
+                                contentDescription = null,
+                                tint = Color.Unspecified,
+                                modifier = Modifier.align(Alignment.CenterStart)
+                                    .padding(start = 11.dp)
+                            )
+
+                            Text(
+                                text = "Continue With Apple",
```

Comment:
Just FYI Apple sign in will never work on Android - so this will need to be hidden later.

Reply:
I'll remove this in a later PR